docs/specs/properties.md

```diff
@@ -0,0 +1,139 @@
+# Properties
+
+## Introduction
+
+Properties in Lottie can be animated.
+
+Their structure depends on whether it's animated or not:
+
+| Attribute | Type | Title | Description |
+|-----------|------|-------|-------------|
+| `a`       | {link:values/int-boolean} | Animated | Whether the property is animated |
+| `k`       | | Value or Keyframes | When it's not animated, `k` will contain the value directly. When animated, `k` will be an array of keyframes. |
+
+<h3 id="base-keyframe">Keyframes</h3>
+
+{schema_object:properties/base-keyframe}
+
+If `h` is present and it's 1, you don't need `i` and `o`, as the property will keep the same value
+until the next keyframe.
+
+<h3 id="easing-handle">Keyframe Easing</h3>
+
+Keyframe easing handles  objects with `x` and `y` attributes, which are numbers within 0 and 1.
+
+{schema_object:properties/easing-handle}
+
+For vector properties, these are arrays, with one element
+per dimension so you can have different easing curves per dimension.
+
+They represent a cubic bezier, starting at `[0,0]` and ending at `[1,1]` where
+the value determines the easing function.
+
+The `x` axis represents time, a value of 0 is the time of the current keyframe,
+a value of 1 is the time of the next keyframe.
+
+The `y` axis represents the value interpolation factor, a value of 0
+represents the value at the current keyframe, a value of 1 represents the
+value at the next keyframe.
+
+When you use easing you have two easing handles for the keyframe:
+
+`o` is the "out" handle, and is the first one in the bezier, determines the curve
+as it exits the current keyframe.
+
+
+`i` is the "in" handle, and it's the second one in the bezier, determines the curve
+as it enters the next keyframe.
+
+
+For linear interpolation you'd have
+
+```json
+{
+    "o": {"x": [0, 0], "y": [0, 0]},
+    "i": {"x": [1, 1], "y": [1, 1]}
+}
+```
+
+For easing in and out, you move the `x` towards the center, this makes the animation more fluid:
+
+```json
+{
+    "o": {"x": [0.333, 0.333], "y": [0, 0]},
+    "i": {"x": [0.667, 0.667], "y": [1, 1]}
+}
+```
+
+<h4>Easing example</h4>
+In the following example, the ball moves left and right, on the background you can see and edit a representation of its easing function.
+
+{editor_example:easing}
+
+
+## Property types
```

Comment:
Ordering: is there a reason for the current order?  If not, it might be helpful to reorganize based on complexity: scalar -> position -> color -> vector -> path.

Reply:
I think position/color should go after vector since they are special cases of vector props.

Reply:
also, scalar properties use vector keyframes, so maybe the order should be something like vector, scalar, position, color, path?